.github/workflows/release.yml

```diff
@@ -168,4 +168,15 @@ jobs:
         uses: actions/github-script@v4
         with:
```

Comment:
We need to supply github-token. branchlet token should do, but that is internal token, so I recommend we create a new token that can have only access to our public repos